scripts/changelog/index.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-env node */
-const fs = require('fs');
+const fs = require('fs').promises;
```

Comment:
Is this necessary? Pretty sure the existing code doesn't use a promise here

Reply:
i was thinking that may solve a error i was having.

Reply:
Fixed!